### DISCUSSION TITLE
【法律求助】我叫来家里的保洁打伤了三个入室抢劫犯

### DISCUSSION THREAD

**Discussion by AAAzhaoxiangji** [share121/inter-knot#2672] [其他]
![561dc532747e7b39babfe35a83b321b](https://github.com/user-attachments/assets/8c1b6fc6-3973-4a34-bb31-f48592bb33cc)
今天我约了一个上门保洁服务。保洁人员因为堵车修路迟到了五分钟，一直跟我道歉本来有点生气，但看到她态度那么卑微，有点不忍心，就算了。
我交代好保洁要求后就去看电影了。看到一半，治安局打电话来，说有三个人闯入我家抢劫。我想起家里还有保洁人员，还是个小姑娘，就问了下她还好吗，治安局说是那个抢劫犯报的秦她把三个壮年男子打进了医院急诊…
现在三个人找律师向我索赔医药费，有懂法的朋友说说我该怎么办吗?

**Comment by HKLHaoBin** [comment]
### 针对您描述的情况，以下是我作为律师提供的法律分析和建议：

**一、法律分析**

1.正当防卫的认定：

保洁人员在您家中遭遇入室抢劫时，为了保护自身安全和财产权益，对抢劫犯实施了防卫行为。根据《中华人民共和国刑法》第二十条规定，为了使国家、公共利益、本人或者他人的人身、财产和其他权利免受正在进行的不法侵害，而采取的制止不法侵害的行为，对不法侵害人造成损害的，属于正当防卫，不负刑事责任。
关键在于判断保洁人员的防卫行为是否超过必要限度。如果防卫行为没有明显超过必要限度造成重大损害，那么应当认定为正当防卫。

2.民事责任的考量：

抢劫犯因受伤而要求赔偿医药费，这涉及民事责任问题。在正当防卫的场合，防卫人通常不需要承担民事责任。但如果防卫行为超过必要限度造成不应有的损害，防卫人应当承担适当的民事责任。
鉴于保洁人员是在保护您和您财产的过程中受伤的抢劫犯，且其防卫行为很可能被认定为正当防卫，因此您作为雇主一般不需要承担民事责任。

**二、建议行动**

1.收集证据：

尽可能收集与案件有关的证据，包括治安局的报警记录、现场勘查报告、医院诊断证明、保洁人员的陈述等。这些证据有助于证明抢劫犯的不法行为和保洁人员的正当防卫。

2.咨询专业律师：

聘请专业律师为您提供法律帮助。律师可以根据具体情况分析案件性质、评估法律风险，并制定相应的应对策略。

3.配合调查：

积极配合治安局的调查工作，如实陈述案件事实。同时，与律师保持密切沟通，确保自身权益不受侵害。

4.关注法律程序：

关注抢劫犯及其律师可能提起的民事诉讼或刑事诉讼程序。在必要时，您可以通过法律途径维护自身权益。

**三、注意事项**

避免与抢劫犯或其律师私下接触或协商赔偿事宜，以免陷入不必要的法律纠纷。
保持冷静和理性，不要因情绪激动而做出过激行为或言论。
遵守法律程序，尊重司法判决结果。

总之，针对您所描述的情况，保洁人员的防卫行为很可能被认定为正当防卫，您作为雇主一般不需要承担民事责任。但为了确保自身权益不受侵害，建议您采取上述行动并咨询专业律师的意见。

<img src="https://img-static.mihoyo.com/communityweb/upload/54f6c5f6a4bf851877158b0000e636c5.png" alt="克拉拉 给你心心" width="105" >

**Reply by Fintbo** [reply]
你是一袋......欸，你是一袋是什么来着?

**Comment by qqqqqq77777** [comment]
你可要小心一点了，他们是不是合伙仙人跳你阿....<img src="https://bbs-static.miyoushe.com/static/2023/11/16/2544d70667e060623d4b7702cd6bb3b9_8763505029750571668.png" alt="可琳-紧张" width="75" >害怕

**Comment by anchangfeng** [comment]
我方难道没有理由认为这三个人涉嫌非法闯入民宅和对他人进行诽谤勒索吗？有证据实锤的话还可以私了敲他们一笔。
